backend/tests/services/kanban/Jira/JiraVerifyToken.test.ts

```diff
@@ -216,8 +196,187 @@ describe("verify token and return columns and users", async () => {
         flag: false,
       },
     ];
-    expectedResponse.users = jiraUserNames;
-    expectedResponse.jiraColumns = jiraColumnNames;
+    const expectedResponse = {
+      jiraColumns,
+      users,
+      targetFields,
+    };
+    const response = await jiraVerifyToken.verifyTokenAndGetColumnsAndUser(
+      tokenVerifyModel
+    );
+    expect(response.targetFields[0].flag).to.equal(false);
+    expect(response.users.length).to.equal(3);
     expect(response).deep.equal(expectedResponse);
   });
 });
+
+describe("get query status", () => {
```

Comment:
should have async for the describe, coc the sub tests have **async**

The below need to be updated